tests/integration/targets/iso_customize/tasks/iso_customize.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Add a line to the file test02.cfg and make sure it succeed
+  ansible.builtin.lineinfile:
+    path: "{{ test_dir }}/test02.cfg"
+    regexp: "^test"
+    line: "test"
+
+- name: "Customize ISO file: add file, delete file and change file"
+  community.general.iso_customize:
+    src_iso: "{{ test_dir }}/test.iso"
+    dest_iso: "{{ test_dir }}/{{ dest_iso_name }}"
+    delete_files:
+      - "/test01.cfg"
+    add_files:
+      - src_file: "{{ test_dir }}/test01.cfg"
+        dest_file: "/preseed/ubuntu.seed"
```

Comment:
Given the discussion in #5283 , it might be interesting to guarantee that we can refer to files inside the ISO image both with or without leading slash.

Reply:
Fix it.
without leading slash in files iso_customize_add_files.yml and leading iso_customize_delete_files.yml
with leading slash in file iso_customize.yml